integration-cli/docker_api_swarm_test.go

```diff
@@ -1002,3 +1003,25 @@ func (s *DockerSwarmSuite) TestSwarmRepeatedRootRotation(c *check.C) {
 		currentTrustRoot = clusterTLSInfo.TrustRoot
 	}
 }
+
+func (s *DockerSwarmSuite) TestAPINetworkInspectWithScope(c *check.C) {
+	d := s.AddDaemon(c, true, true)
+
+	out, err := d.Cmd("network", "create", "-d", "overlay", "foo")
+	c.Assert(err, checker.IsNil, check.Commentf(out))
```

Comment:
Not sure about this. I think we probably want to create using an API call as well. Maybe it's fine for now. 

Using the docker client should make it just as easy to create the network.